example_buildpack_test.go

```diff
@@ -0,0 +1,44 @@
+package libdependency_test
+
+import (
+	"path/filepath"
+
+	"github.com/Masterminds/semver/v3"
+	"github.com/joshuatcasey/libdependency"
+	"github.com/joshuatcasey/libdependency/versionology"
+)
+
+func ExampleGetNewVersionsForId() {
```

Comment:
Can you add a comment explaining what this is for? Is it used in a test somewhere?

Reply:
It's a testable example for the STDOUT. https://go.dev/blog/examples